examples/previous-next-navigation/static.config.js

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios'
+
+export default {
+  getSiteData: () => ({
+    title: 'React Static',
+  }),
+  getRoutes: async () => {
+    const { data: posts } = await axios.get('https://jsonplaceholder.typicode.com/posts')
+    return [
+      {
+        path: '/',
+        component: 'src/containers/Home',
+      },
```

Comment:
If the `Home.js` component is moved to `pages/index.js`, then this route definition isn't needed anymore 😄.

Reply:
Done